pkg/streamingpromql/query.go

```diff
@@ -310,10 +313,11 @@ func (q *Query) convertToInstantVectorOperator(expr parser.Expr, timeRange types
 }
 
 func (q *Query) convertFunctionCallToInstantVectorOperator(e *parser.Call, timeRange types.QueryTimeRange) (types.InstantVectorOperator, error) {
```

Comment:
Do we need to do the same thing for functions over range vectors?

Reply:
Function over range vectors are done here. The output is just instant vectors.

Reply:
🤦 Of course they are, ignore me 🙂